workflow.txt

```diff
@@ -8,3 +8,15 @@ git merge my-solution
 fork on github
 git checkout master
 git push origin my-solution
+
+
+fork on gitbub
+git clone
+git fetch -all
+git checkout -b my-solution
```

Comment:
All good up to here. After checking out our new branch we want to add and commit our changes.
`git add .`
`git commit -m ""`
then we can switch to `master`
`git checkout master`
pull any recent changes
`git pull upstream master`
finally merge our branch into master and push
`git merge my-solution`
`git push origin my-solution`